azure-pipelines.yml

```diff
@@ -46,6 +53,11 @@ jobs:
       name: linux
       vmImage: ubuntu-16.04
       matrix:
+        py_3.5_32:
+          MB_PYTHON_VERSION: "3.5"
+          PLAT: "i686"
+        py_3.5_64:
+          MB_PYTHON_VERSION: "3.5"
```

Comment:
I think we should force the build against the oldest supported of numpy for Python 3.5.

Reply:
Yes but the existing ones are not following oldest supported numpy. I thought that was intentional

Reply:
It's probably because I used the version numbers for 0.23 and not 0.22.X

Reply:
Anyway that should be fine as it is.